.github/workflows/find-issues-with-missing-labels.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Find issues with missing labels'
+
+on:
+  schedule:
+    # Run every Sunday at midnight
+    - cron: '0 0 * * 0'
+
+permissions:
+  issues: write
```

Comment:
Do not use top level permissions these are reducing the OSSF score

Reply:
updated to permission on job